cmd/utils/flags.go

```diff
@@ -218,7 +218,17 @@ var (
 	}
 	EtherbaseFlag = cli.StringFlag{
```

Comment:
How does this flag differ from `MinerLegacyEtherbaseFlag`? It seems to me like there might have been a bad merge here. I dont see this flag used in this file.

Reply:
I think you're right, removed.